Bio/PDB/Atom.py

```diff
@@ -16,14 +16,17 @@
 
 
 class Atom(object):
+    """Define Atom class.
+
+    The Atom object stores atom name (both with and without spaces),
+    coordinates, B factor, occupancy, alternative location specifier
+    and (optionally) anisotropic B factor and standard deviations of
+    B factor and positions.
+    """
+
     def __init__(self, name, coord, bfactor, occupancy, altloc, fullname, serial_number,
                  element=None):
-        """Create Atom object.
-
-        The Atom object stores atom name (both with and without spaces),
-        coordinates, B factor, occupancy, alternative location specifier
-        and (optionally) anisotropic B factor and standard deviations of
-        B factor and positions.
+        """Initialize Atom object.
```

Comment:
Why did you remove the four lines of explanation here?

Reply:
I moved it from __init__ to the class definition of the __init__, Atom; because the class Atom did not have a DocString, the __init__ is just below the class definition, and the content seem good to encompass the class. I am happy to put it back in the __init__ and leave the class DocString as a one-liner; or to have it on both, looks a bit redundant as the __init__ is just below; happy for suggestions,

Reply:
Oh yeah - I must have missed that - good choice.